.circleci/config.yml

```diff
@@ -623,9 +620,10 @@ workflows:
     jobs:
       - danger
   weekly-run-workflow:
-    when:
-      and:
-        - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
-        - equal: [ "release-train", << pipeline.schedule.name >> ]
+    # when:
+      # and:
+        # - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+        # - equal: [ "release-train", << pipeline.schedule.name >> ]
```

Comment:
Probably unintentional to leave this in, right?

Reply:
oh yes, I just commented this to trigger it the job lol, I wanna see if it works as expected

Reply:
I actually think there's a way of running this pipeline from CircleCI. Next time. The job is almost done I don't want to cancel it

Reply:
actually, let's merge it and run it on main